java/server/src/org/openqa/selenium/grid/sessionmap/SessionMap.java

```diff
@@ -81,6 +81,10 @@
 
   public abstract void remove(SessionId id);
 
+  public int getCount() {
```

Comment:
This is not the right approach. The `Distributor` maintains a model of the current state of the Grid. That model already contains the information about every active session. We don't need to modify `SessionMap` to expose it further.